src/main/java/uk/gov/hmcts/darts/common/entity/EventEntity.java

```diff
@@ -87,4 +87,9 @@ public class EventEntity extends CreatedModifiedBaseEntity {
     public void addHearing(HearingEntity hearingEntity) {
         hearingEntities.add(hearingEntity);
     }
-}
+
+    public HearingEntity associatedHearing() {
```

Comment:
author note:
I'm not using the standard `get*` notation because I don't want a `hearing` property to be added to the entity (and potentially show up in endpoints JSON responses). This is more an utility method

Reply:
I think we can still use the get* convention here as we should never return Spring Entity as responses anyway.
We could even add a @JsonIgnore annotation to ensure it does not get added to Json output

Reply:
The JsonIgnore annotation would prevent this from showing up in a Json Response, and as we are getting the hearing entity I still think the standard naming convention of getHearing() makes sense here?

Reply:
Not really bothered either way but not sure what we mean by _potentially show up in endpoints JSON_? We have mappers to address this? Entity content shouldnt just be automatically added to the json response?